src/person/simple_serializers.py

```diff
@@ -19,6 +19,16 @@ class SimpleUserSerializer(serializers.ModelSerializer):
     interests = InterestSerializer(source="person.interests", many=True)
     groups = GroupSerializer(source="person.groups", many=True)
     prompts = serializers.SerializerMethodField("get_prompts")
+    pending_feedback = serializers.BooleanField(source="person.pending_feedback")
+    blocked = serializers.SerializerMethodField("get_blocked")
```

Comment:
`blocked_user_id`?

Reply:
its a boolean actually, not an id
it describes whether or not that user is blocked by the currently-authenticated user

from my overview above: I have added a boolean field named blocked on the community user model (i.e. GET /api/users/<id>/). It returns true if the current user blocked the other user, and false otherwise.